src/super_gradients/training/models/detection_models/csp_darknet53.py

```diff
@@ -232,9 +274,9 @@ def __init__(self, arch_params: HpmStruct):
         struct = [depth_mult(s) for s in struct]
         self._modules_list = nn.ModuleList()
 
-        if get_param(arch_params, 'stem_type') == 'focus' or yolo_type == 'yoloX' or yolo_version == 'v3.0':
+        if get_param(arch_params, 'stem_type') == 'focus' or yolo_version == 'v3.0':
             self._modules_list.append(Focus(channels_in, width_mult(64), 3, 1, activation_type))  # 0
-        elif get_param(arch_params, 'stem_type') == '6x6' or yolo_version == 'v6.0':
+        elif get_param(arch_params, 'stem_type') == '6x6' or yolo_type == 'yoloX' or yolo_version == 'v6.0':
```

Comment:
Will break yolox weights, which aren't among pretrained models tests yet, but just so you know when you add them. Let's discuss offline how to reshape them into 6x6.